cmd/metacache-set.go

```diff
@@ -606,8 +606,8 @@ func (er *erasureObjects) listPath(ctx context.Context, o listPathOptions, resul
 	defer close(results)
 	o.debugf(color.Green("listPath:")+" with options: %#v", o)
 
-	// get non-healing disks for listing
-	disks, _ := er.getOnlineDisksWithHealing()
+	// get prioritized non-healing disks for listing
+	disks, _ := er.getOnlineDisksWithHealing(true)
```

Comment:
Do we want healing disks to be included? That only has partial content, anyway? - it will sure tax that drive more IMO.

Reply:
Actually I see there is benefit in it.